app/src/main/java/com/example/android/mashup/CreatorAudio/CreatorChooseAudioFragment.kt

```diff
@@ -69,8 +69,15 @@ class CreatorChooseAudioFragment : Fragment(), MashupClickListener {
             GetVideoDataForUris(audioUriViewModel.readAllData.value), this
         );
 
+        val orientation = resources.configuration.orientation
+        val span = if (orientation == Configuration.ORIENTATION_LANDSCAPE){
```

Comment:
Can be replaced with `orientation == Configuration.ORIENTATION_LANDSCAPE ? 2 : 1`